mmv1/products/cloudrun/terraform.yaml

```diff
@@ -316,6 +316,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         skip_docs: true
         vars:
           cloud_run_srv: "cloud-run-srv"
+        min_version: beta
```

Comment:
This marks the previous example as beta-only. Is there a reason to do that?

Reply:
Not sure how it got there. I removed this line